QuasarHelios2015/src/org/team1540/quasarhelios/DriveCode.java

```diff
@@ -144,9 +144,17 @@ public static void setup() {
         centricAngleOffset = ControlInterface.mainTuning.getFloat("main-drive-centricAngle", 0);
         recalibrateButton.send(calibrate);
 
-        FloatStatus p = ControlInterface.mainTuning.getFloat("main-drive-p", 0.01f);
-        FloatStatus i = ControlInterface.mainTuning.getFloat("main-drive-i", 0f);
-        FloatStatus d = ControlInterface.mainTuning.getFloat("main-drive-d", 0f);
+        FloatStatus ultgain = ControlInterface.mainTuning.getFloat("drive-PID-ultimate-gain", .0162f);
+        FloatStatus period = ControlInterface.mainTuning.getFloat("drive-PID-oscillation-period", 2f);
+        BooleanStatus calibrating = ControlInterface.mainTuning.getBoolean("calibrating-drive-PID", false);
+
+        FloatInput p = Mixing.select(calibrating, FloatMixing.multiplication.of((FloatInput) ultgain, .6f), ultgain);
```

Comment:
0.6f sounds like a magic number. Either it should be a named constant or should be tunable.